include/structures/junction.hpp

```diff
@@ -29,8 +30,10 @@ class Junction
     Junction(Breakend the_mate1,
              Breakend the_mate2,
              auto const & the_inserted_sequence,
+             int16_t const & the_tandem_dup_count,
              std::string the_read_name) : mate1{std::move(the_mate1)},
                                           mate2{std::move(the_mate2)},
+                                          tandem_dup_count{std::move(the_tandem_dup_count)},
```

Comment:
integer are not movable. So it will copy anyways.
```suggestion
                                          tandem_dup_count{the_tandem_dup_count},
```